lib/tasks/user.rake

```diff
@@ -1,12 +1,21 @@
 # frozen_string_literal: true
 
 namespace :user do
-  desc 'Updates the first_admin_id attribute value for all providers.'
+  desc 'Updates the first_admin_id attribute value for all accounts.'
   task update_all_first_admin_id: :environment do
-    Account.transaction do
```

Comment:
This transaction should not have been done. If we have 1M of accounts it will only release the process after everything is done and it is not the right thing to do. And the worse is that it will create a row lock for all accounts. :/